src/main/java/sirius/kernel/xml/Outcall.java

```diff
@@ -92,6 +89,18 @@
      */
     private static final int TIMEOUT_BLACKLIST_HIGH_WATERMARK = 100;
 
+    @ConfigValue("http.outcall.defaultConnectTimeout")
```

Comment:
Isn' this a bit of an overkill here? We can already overwrite the default timeouts if needed...

Reply:
The idea was to keep the existing defaults while being able to configure them accordingly in the dependant products systemwide. Therefore you'll not have to call `setConnectTimeout` or `setReadTimeout` before every request.